<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator_Form</class>
 <widget class="QDialog" name="Calculator_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="dialog_horizontal_layout">
   <item>
    <layout class="QVBoxLayout" name="settings_vertical_layout">
     <item>
      <widget class="QFrame" name="window_settings_frame">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>180</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>180</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="y_axis_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>50</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>y-axis:</string>
        </property>
       </widget>
       <widget class="QLabel" name="x_axis_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>50</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>x-axis:</string>
        </property>
       </widget>
       <widget class="QLabel" name="window_settings_label">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>10</y>
          <width>81</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Window:</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>315</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="x_axis_settings_layout">
         <item>
          <widget class="QCheckBox" name="x_axis_enabled_check_box">
           <property name="text">
            <string>Enabled</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="x_axis_markers_enabled_check_box">
           <property name="text">
            <string>Show Markers</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="x_axis_lower_bound_line_edit">
           <property name="text">
            <string>-10</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="x_axis_range_label">
           <property name="text">
            <string>≤ x ≤</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="x_axis_upper_bound_line_edit">
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>314</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="y_axis_settings_layout">
         <item>
          <widget class="QCheckBox" name="y_axis_enabled_check_box">
           <property name="text">
            <string>Enabled</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="y_axis_markers_enabled_check_box">
           <property name="text">
            <string>Show Markers</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="y_axis_lower_bound_line_edit">
           <property name="text">
            <string>-10</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="y_axis_range_label">
           <property name="text">
            <string>≤ y ≤</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="y_axis_upper_bound_line_edit">
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="function_table_widget">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>170</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>170</height>
        </size>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <row>
        <property name="text">
         <string>1)</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2)</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>3)</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>4)</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>5)</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Input</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Output</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Color</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="button_grid_layout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <item row="2" column="0">
        <widget class="QPushButton" name="reset_graph_push_button">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="update_graph_push_button">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="remove_function_push_button">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>Remove Function</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="add_function_push_button">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>Add Function</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="vertical_spacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
